docker/kubernetes/README.md

```diff
@@ -0,0 +1,234 @@
+
+`$ kubectl create configmap node-registry --from-file node-registry.yaml`
+
+Verify the ConfigMaps were created:
+
+```
+$ kubectl get cm
+NAME            DATA   AGE
+key-registry    1      30s
+node-registry   1      2m43s
+```
+
+You can inspect the values of the ConfigMaps by running
+`kubectl describe cm <configmapname>`. For example:
+
+```
+$ kubectl describe cm node-registry
+Name:         node-registry
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+node-registry.yaml:
+----
+---
+- identity: "acme"
+  metadata:
+    organization: "Acme Corporation"
+    endpoint: "tls://acme.default.svc.cluster.local:8044"
+- identity: "bubba"
+  metadata:
+    organization: "Bubba Bakery"
+    endpoint: "tls://bubba.default.svc.cluster.local:8044"
+
+Events:  <none>
+```
+
+### Step 4: Starting Gameroom
+Apply the `arcade.yaml` manifest:
+
+```
+$ kubectl apply -f https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/arcade.yaml
+
+deployment.apps/acme created
+service/acme-splinterd created
+service/acme-http created
+deployment.apps/bubba created
+service/bubba-splinterd created
+service/bubba-http created
+```
+
+Verify the pods started correctly:
+
+```
+$ kubectl get pods
+NAME                     READY   STATUS      RESTARTS   AGE
+acme-7575f75d6d-trddh    5/5     Running     0          6m10s
+bubba-5d5f554fdb-l9nr4   5/5     Running     0          6m9s
+gameroom-keys-l42fz      0/1     Completed   0          56m
+```
+
+### Step 5: Create Users and Log In to the Web App
+
+Run `minikube service` to open the Acme Web App.
+
+```
+$ minikube service acme-http
+|-----------|-----------|-------------|---------------------------|
+| NAMESPACE |   NAME    | TARGET PORT |            URL            |
+|-----------|-----------|-------------|---------------------------|
+| default   | acme-http | http        | http://192.168.64.3:30160 |
+|-----------|-----------|-------------|---------------------------|
+Opening service default/acme-http in default browser...
+```
+![alt text](images/acme-1-launch.png "Acme gameroom homepage")
+
+In the upper left, click "Register".
+
+Fill out the registration form with values you'll remember. For the
+`Private Key` field, enter the private key information generated for Alice in
+Step 1.
+
+![alt text](images/acme-2-register.png "Acme registration page")
+
+You're logged in as Alice now.
+![alt text](images/acme-3-loggedin.png "Alice logged in page")
+
+Repeast steps for Bubba/Bob.
+
+`$ minikube service bubba-http`
+
+![alt text](images/bubba-1-launch.png "Bubba gameroom homepage")
+
+![alt text](images/bubba-2-register.png "Bubba registration page")
+
+![alt text](images/bubba-3-loggedin.png "Bob logged in page")
+
+### Step 6: Create a gameroom
+
+Switch back to the Acme instance. Click the `+` next to `My Gamerooms`.
+Select Bubba from the dropdown menu and give your new gameroom a name.
+Then click Send.
```

Comment:
Be consistent with web app buttons and labels. Above, "Register" is surrounded with quotes; here, "Send" has nothing; and below, "Invitations" and "SENT" are in fixed-width font. 

FYI, the doc convention is to use bold for all UI labels & buttons: **Register**, **Send**, **Invitations**, **SENT**.